tests/fpga_system_tests/emu/main.c

```diff
@@ -25,6 +25,9 @@ void do_init() {
    set_jitter_rms_int(0);
    set_noise_rms_int(0);
 
+   // prbs control
+   set_prbs_eqn(0x100002);
```

Comment:
the PRBS equation can now be changed at runtime.  some of the shorter PRBS sequences, like PRBS-7, have a very clearly apparent repetition that distorts the BER